docs/contributors/releasing.md

```diff
@@ -112,18 +119,24 @@ title: Making a release
   - Once the VS Code extension has been updated on the Marketplace, click
     "Publish release".
 
-- Update version numbers in:
-  - `build.sbt` - update `localSnapshotVersion`
```

Comment:
Why remove this? This should be done during the release ideally.

Reply:
It's duplicated, see lines 49-53

Reply:
However, this one is more concise